golang/lib/networks/network_context.go

```diff
@@ -38,11 +38,9 @@ Creates a new NetworkContext object with the given parameters.
 */
 func NewNetworkContext(
 	client kurtosis_core_rpc_api_bindings.ApiContainerServiceClient,
-	filesArtifactUrls map[services.FilesArtifactID]string,
```

Comment:
With this gone, NetworkContext is now truly stateless!!!

Reply:
👍